README.md

```diff
@@ -149,28 +153,28 @@ type  │                    data
      format
 ```
 
-### Encrypted data formats
+### 5. encrypted formats
 
 When content is encrypted (in other words, "boxed") in SSB, it is provided as
 uninterpretable bytes, plus a tag that identifies which algorithm was used for
 encrypting it, such as `box` or `box2`.
 
 | Type code | Format code | Data length | Format name | Specification   |
-|-----------|-------------|-------------|-------------|-----------------|
-| 5         | 0           | Arbitrary   | box         | [private box]   |
+|:---------:|:-----------:|-------------|-------------|-----------------|
+| 5         | 0           | Arbitrary   | box1        | [private box]   |
```

Comment:
This one still confuses me, which one is it, box1 or box? I honestly don't know. I know that once code broke everywhere because we were using `box1` and it had to be `box`. But I think (??) ssb-tribes uses `box1`.

Reply:
Ssb tribes uses box2
The original is box, but since the advent of box2 we talked about box saying "box1"... but the suffix is the OG box

At least I think that's how it went down